pkgs/development/python-modules/pdfminer-six/default.nix

```diff
@@ -2,53 +2,41 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
-  fetchpatch2,
   cryptography,
   charset-normalizer,
   pythonOlder,
   pytestCheckHook,
   setuptools,
-  replaceVars,
+  setuptools-scm,
   ocrmypdf,
 }:
 
 buildPythonPackage rec {
   pname = "pdfminer-six";
-  version = "20240706";
+  version = "20250506";
   pyproject = true;
-
   disabled = pythonOlder "3.8";
```

Comment:
```suggestion
```
we don't support such old versions anyway

Reply:
```
$ rg 'pythonOlder "3.[1-9]"' -l | wc -l
3715
```

rather than fix these one review cycle at at time i invite you to consider a treewide